README.md

```diff
@@ -1,3 +1,7 @@
+Current Build Status
+--------------------
+
+[![Build Status](https://travis-ci.org/leider/CoolBeans.png)](https://travis-ci.org/leider/CoolBeans)
```

Comment:
I just looked again and see that this doesn't reference anything at Github. I imagine your Travis-CL config references your fork? I'd think that we can keep this Travis setup but just point to my main repo.